scss/print.scss

```diff
@@ -23,26 +23,26 @@
   .app-layout__note-column,
   .note-editor,
   .note-detail-wrapper,
-  .note-detail {
-    display: block;
-    overflow: visible;
-  }
-
+  .note-detail,
+  .note-detail-textarea,
+  .note-content-editor-shell,
   .note-detail-wrapper {
     overflow: visible;
   }
 
-  .note-detail {
-    overflow: visible;
-  }
-
   [class^='note-detail-'] {
     max-width: 100%;
     width: 100%;
     color: $studio-black;
 
     &.theme-color-fg {
       color: $studio-black;
+      overflow-y: auto !important;
     }
   }
+
+  .note-content-plaintext {
+    display: inherit;
+    white-space: pre-wrap;
```

Comment:
why do we have `white-space` added here and on `&.visible` in `theme.scss` instead of on the root `.note-content-plaintext` rule? are we taking it out of the hidden view for some side-effect?